src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -524,6 +546,45 @@ export class StudyViewPageStore {
         this.notifyComparisonGroupsChange();
     }
 
+    public checkSelectedGroupsColors(groupUid: string, color: string) {
```

Comment:
Can you write one or two lines of comment describing the goal of the code below?